bin/arno-iptables-firewall

```diff
@@ -442,7 +442,7 @@ setup_kernel_settings()
   #############################################################
   if sysctl_key_match net.netfilter.nf_conntrack_helper; then
     sysctl -w net.netfilter.nf_conntrack_helper=0
-  else
+  elif ! kernel_ver_chk 5 4 212; then
```

Comment:
``elif ! kernel_ver_chk 6 0 0; then``

seems better to me

Reply:
Then later at line: 803
```
if [ "$(sysctl_get_value net.netfilter.nf_conntrack_helper)" = "0" ]; then
```
change to:
```
if kernel_ver_chk 6 0 0 || [ "$(sysctl_get_value net.netfilter.nf_conntrack_helper)" = "0" ]; then
```

Reply:
> `elif ! kernel_ver_chk 6 0 0; then`
> 
> seems better to me

But what about https://lwn.net/Articles/907206/ ?

Reply:
> Then later at line: 803
> 
> ```
> if [ "$(sysctl_get_value net.netfilter.nf_conntrack_helper)" = "0" ]; then
> ```
> 
> change to:
> 
> ```
> if kernel_ver_chk 6 0 0 || [ "$(sysctl_get_value net.netfilter.nf_conntrack_helper)" = "0" ]; then
> ```

Thanks for the heads up. I totally forgot about that one...

Reply:
> But what about https://lwn.net/Articles/907206/ ? Looks like it got backported?

Hmmm, I'm not seeing the backport commit in 5.4.212.

Here is ``5.4.233`` and the sysctl is still there:

https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/net/netfilter/nf_conntrack_helper.c?h=v5.4.233#n38

I doubt this sysctl removal will be backported to earlier than 6.0, IMO.

BTW, while we could use ``elif ! kernel_ver_chk 3 5 0; then`` (after a lot of research) and that should also work, it just seems easier to understand when consistently comparing with 6.0.0, as that is where the change took place.

Reply:
@abelbeck : Updated. Please review once again and hit green if ok. Thanks!

Reply:
Done.